project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=1.0.0-RC2
```

Comment:
1.0.2 is out

Reply:
Thanks, I should definitely upgrade. As a more general update, still blocked on scala-native (last checked last week)

Reply:
Yeah. Though I see that https://github.com/scala-native/scala-native/issues/915 is now labelled as "hot" so hopefully that means "high priority".

Reply:
Yeah, seen that :) Hopefully it'll be available soon